compute/cloud-client/src/main/java/compute/custommachinetype/CreateCustomMachineType.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.custommachinetype;
+
+// [START compute_custom_machine_type_create]
+
+import com.google.cloud.compute.v1.AttachedDisk;
+import com.google.cloud.compute.v1.AttachedDiskInitializeParams;
+import com.google.cloud.compute.v1.InsertInstanceRequest;
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.NetworkInterface;
+import com.google.cloud.compute.v1.Operation;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateCustomMachineType {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String projectId = "your-google-cloud-project-id";
+    // Name of the zone to create the instance in. For example: "us-west3-b".
+    String zone = "gcloud-zone";
+    // Name of the new virtual machine (VM) instance.
+    String instanceName = "instance-name";
+    // Machine type of the VM being created. This value uses the
+    // following format: "zones/{zone}/machineTypes/{typeName}".
+    // For example: "zones/europe-west3-c/machineTypes/f1-micro"
+    // OR
+    // It can be a CustomMachineType object, describing a custom type you want to use.
+    String machineType = "zones/{zone}/machineTypes/{typeName}";
```

Comment:
Is there not a helper method for creating this string?

Reply:
Yes, there is a helper method.
But the setMachineType() accepts only a string and hence I thought this might be an easier approach, rather than creating a machineType and wrapping it to string.